Specs/DateTime/English/DatePeriodExtractor.json

```diff
@@ -3717,5 +3717,29 @@
         "Length": 17
       }
     ]
+  },
+  {
+    "Input": "I will do my work between now and November 15th",
+    "NotSupported": "python, javascript",
```

Comment:
If you added a Python implementation, why are these cases marked as not supported in it?